model/src/main/java/org/projectnessie/api/params/ReferencesParams.java

```diff
@@ -67,18 +68,22 @@
   public ReferencesParams() {}
 
   private ReferencesParams(
-      Integer maxRecords, String pageToken, boolean fetchAdditionalInfo, String filter) {
+      Integer maxRecords, String pageToken, FetchOption fetchOption, String filter) {
     super(maxRecords, pageToken);
-    this.fetchAdditionalInfo = fetchAdditionalInfo;
+    this.fetchOption = fetchOption;
     this.filter = filter;
   }
 
   private ReferencesParams(Builder builder) {
-    this(builder.maxRecords, builder.pageToken, builder.fetchAdditionalInfo, builder.filter);
+    this(builder.maxRecords, builder.pageToken, builder.fetchOption, builder.filter);
+  }
+
+  public FetchOption fetchOption() {
+    return fetchOption;
   }
 
-  public boolean isFetchAdditionalInfo() {
-    return fetchAdditionalInfo;
+  public boolean isFetchAll() {
```

Comment:
Still there

Reply:
don't know how I forgot to remove that...thanks for spotting. removed it